tensorflow/python/ops/math_ops.py

```diff
@@ -3290,29 +3305,46 @@ def cumsum(x, axis=0, exclusive=False, reverse=False, name=None):
   element of the input is identical to the first element of the output:
 
   ```python
-  tf.cumsum([a, b, c])  # [a, a + b, a + b + c]
+  # tf.cumsum([a, b, c])   # [a, a + b, a + b + c]
+  >>> x = tf.constant([2, 4, 6, 8])
+  >>> tf.cumsum(x)    # [ 2,  6, 12, 20]
+  
+  # using varying `axis` values
+  >>> y = tf.constant([[2, 4, 6, 8], [1,3,5,7]])
+
+  >>> tf.cumsum(y, axis=0)    # [[ 2,  4,  6,  8],
+                              #  [ 3,  7, 11, 15]]
+
+  >>> tf.cumsum(y, axis=1)    # [[ 2,  6, 12, 20],
+                              #  [ 1,  4,  9, 16]]
   ```
 
   By setting the `exclusive` kwarg to `True`, an exclusive cumsum is performed
   instead:
 
   ```python
-  tf.cumsum([a, b, c], exclusive=True)  # [0, a, a + b]
+  # tf.cumsum([a, b, c], exclusive=True)  => [0, a, a + b]
+  >>> x = tf.constant([2, 4, 6, 8])
+  >>> tf.cumsum(x, exclusive=True)  # [ 0,  2,  6, 12]
```

Comment:
The output should be below.

```
>>> tf.cumsum(x, exclusive=True)
Output goes here
```

Please see: https://www.tensorflow.org/community/contribute/docs_ref#docstring_considerations